tests/expression_statement/jsfmt.spec.js

```diff
@@ -1,2 +1 @@
-// TODO: Re-enable Flow when the following fix is merged facebook/flow#3234.
-run_spec(__dirname, { parser: "babylon" } /*, ["typescript"] */);
+run_spec(__dirname, { parser: "babylon" }, ["flow"]);
```

Comment:
Flow parses this now correctly. But the tests for flow succeeds although when I test manually flow also does the wrong thing here and removes the parens. Not sure why the test is succeeding for flow.